JM/CF/Debugger/3_Game/CF_Debugger/CF_Debugger.c

```diff
@@ -0,0 +1,199 @@
+/* Example:
+
+void EOnSimulate(...)
+{
+	auto dbg_Helicopters = CF.Debugger.Get("Helicopters", this);
```

Comment:
I am unsure what this returns. Does it return the block called helicopters? if so why also pass in an object instance? Is this some sort of combined GetBlockByName and GetBlockByObjectInstance? If so these should be separate functions.